README.md

```diff
@@ -17,6 +17,7 @@ backends = c.backends.list()
 
 results = c.directories.headers(term='alice', profile='default', token='my-valid-token')
 results = c.directories.lookup(term='alice', profile='default', token='my-valid-token')
+results = c.directories.lookup_user(term='alice', profile='default', xivo_user_uuid='user-uuid', token='my-valid-token')
```

Comment:
remove the `xivo_` prefix on new code

Reply:
With what was done in dird, is this request still valid?